nodethon/README.md

```diff
@@ -0,0 +1,72 @@
+# "Nodethon" (Node.js + Python) Docker
+
+Builds and pushes a docker environment for use with tox testing. This environment contains Node.js 13 and some other JavaScript dependencies, besides a series of python versions allowing multi-version tox testing locally and in CI services.
+
+The "sq" suffix indicates that the ["sonarqube"](https://www.sonarqube.org) scanner has been included in the image. This scanner allows tox builds to upload coverage and other reports to a sonarqube instance.
+
+## Build and Push
+
+These instructions are for maintainers with permissions to push to the "uavcan" organization on Docker Hub.
+
+```
+docker build .
+```
+```
+docker images
+
+REPOSITORY      TAG            IMAGE ID
+nodethon        latest         d7ab132649d6
+```
+```
+# We use the range of python environments supported as the version tag.
+docker tag d7ab132649d6 uavcan/nodethon:py35-py38-node13-sq
```

Comment:
same comment, search and remove all `-sq` suffixes. Let's also include node version here:

```
uavcan/nodethon:node13-py37-py38
```